### PR TITLE
yarn rw prisma commands now show a little tip

### PR DESCRIPTION
**Pull request by dac09**
**Examples:**
1. When running help (either as flag, or command): https://s.tape.sh/EykeM9UH
2. When running a command that tells the user to run prisma directly: https://s.tape.sh/7UcUY1t8
3. When prisma cli errors out (which prisma defaults to their help command): https://s.tape.sh/J8UMGsR4
4. Show what prisma command its actually running: https://s.tape.sh/OJjrYLNx

Bonus:
- prisma cli command is now in TS

### CODE REVIEW

File: packages/cli/src/commands/prisma.ts [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+      autoFlags.push('--schema', `"${paths.api.dbSchema}"`)
+    }
+  }
+
+  // Set prevents duplicate flags
+  const args = Array.from(new Set([...argv, ...autoFlags]))
+
+  console.log(
+    c.green(`Running prisma cli: \n`) +
+      c.info(`yarn prisma ${args.join(' ')} \n`)
```
Comment (dthyresson) [inline packages/cli/src/commands/prisma.ts:79]: Might this also be a color? Green or something that stands out?

**Comment by dthyresson** [comment]
It may be too cute, but I like the inclusion of emoji in command line output. And maybe not all OS's can output it ... but they do standout as vibrant "bullet"-like points and draw ones attention to the line amid the normal stdout of output.

Maybe @kimadeline could suggest some per prisma command?

File: packages/cli/src/commands/prisma.ts (hunk 1 above)
Comment (dac09) [reply packages/cli/src/commands/prisma.ts:79]: Yeah originally had it as colour, would love some more opinoins on it.

**Comment by kimadeline** [comment]
I heard 🐡🧇 _emojis_ 🛰🧼 Thank you for the ping DT! 

Love the Redwood tip! And the TS conversion, obviously 🙂 

My humble suggestions, in no particular order:
- help: ℹ️ 🦺 📖 💡 (there's also 🆘, but it carries a feeling of urgency that may not suit the purpose of this command)
- introspect: 🔍 🔎 🔬 
- error: ❌
- running the CLI: 🏗️ ⏳ 🕥

**Comment by dthyresson** [comment]
BTW I noticed that `prisma db seed` completes with:

```
🌱  Your database has been seeded.
✨  Done in 4.89s.
```

**Comment by thedavidprice** [comment]
Note for future disscussion: Peter and I had some conversation about leaving the CLI package as JS. No need to discuss now nor any reason to not move forward as is. But reminder to us all about finalizing some ongoing discussions about overall approach to TS in the Framework itself.

**Comment by thedavidprice** [comment]
Screen Shot 2021-02-10 at 3 49 26 PM

Am I doing emojies right?

**Comment by thedavidprice** [comment]
Downside of this approach is a .5 sec delay when running command toward the end -- pauses right before displaying this box.

**Comment by cannikin** [comment]
Were we going to mention which flags we're setting? So that when you see them mentioned in the Prisma help output you know they can be safely ignored? `--preview-feature` and `--schema`?

**Comment by dac09** [comment]
The command shows what it's running (and what flags it's passing) right at the top. This tip is just when help is shown, if it errors or if prisma suggests a direct prisma command. 

Wasn't sure how much I should make it standout though

(^check out the tapes in the description)

**Comment by cannikin** [comment]
I think it's the phrase "your project's configuration flags" that's making me feel weird...like `--preview-feature` has nothing to do with my app's config, that's just what's need to get migrate to run at all while in beta! 

But I can't think of anything else to put there instead, without being a lot more verbose:

> You can use the Prisma CLI directly (`yarn prisma`) but letting 
> Redwood act as a proxy (`yarn redwood prisma`) adds a couple 
> of flags and saves you from a lot of extra typing. 
>
> `--preview-feature` and `--schema=api/db/schema.prisma` 
> flags will be added automatically.
>
> To turn off this message add `show_prisma_note = false` in `redwood.toml`

Turning it off is a nice-to-have: I feel like once you read the note the first time you understand, but we're going to keep showing it FOREVER and it's just making the output even more verbose. 🤷‍♂️ 🍔

**Comment by thedavidprice** [comment]
Added names of the options to be more clear. Kept it short.

Screen Shot 2021-02-10 at 6 43 34 PM

100% agreed about inevitably wanted to disable via config. Shipping this as-is for now.

**NEXT STEP**
- [ ] Add feature to turn off this message `show_prisma_note = false` in redwood.toml
    - add to doc info about how to disable

**Pull request merged**